### PR TITLE
Document base web server configuration for GLPI >= 10.0.7

### PR DESCRIPTION
**Pull request by cedric-anne**
Documentation related to https://github.com/glpi-project/glpi/pull/13197

### CODE REVIEW

**Comment by AdrienClairembault** [comment]
I've tried the following configuration for nginx:

```nginx
server {
        listen 8083;

        root /home/adrie/localhost/glpi/10.0-bugfixes/public;

        # Add index.php to the list if you are using PHP
        index index.html index.htm index.nginx-debian.html index.php;

        server_name localhost;
        access_log /var/log/nginx/access.log;
        error_log /var/log/nginx/error.log;

        location / {
                try_files $uri /index.php$is_args$args;
        }

        location ~ ^/index\.php$ {
                fastcgi_pass unix:/run/php/php8.0-fpm.sock;

                fastcgi_split_path_info ^(.+\.php)(/.*)$;
                include fastcgi_params;
                fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
        }
}
```

Going to `http://localhost:8083` redirect me to `http://localhost:8083/glpi/10.0-bugfixes/` which is forbidden.

image

File: source/prerequisites.rst [hunk 1]
```diff
@@ -16,6 +18,65 @@ GLPI requires a web server that supports PHP, like:
 * `Nginx <http://nginx.org/>`_;
 * `Microsoft IIS <http://www.iis.net>`_.
 
+Apache configuration
+^^^^^^^^^^^^^^^^^^^^
+
+Here is a virtual host configuration example for ``Apache 2`` web server using ``mod_php``.
```
Comment (trasher) [inline source/prerequisites.rst:24]:
```suggestion
Here is a virtual host configuration example for ``Apache 2`` web server.
```

**Review by trasher — APPROVED** [review]

File: source/prerequisites.rst [hunk 2]
```diff
@@ -16,6 +18,65 @@ GLPI requires a web server that supports PHP, like:
 * `Nginx <http://nginx.org/>`_;
 * `Microsoft IIS <http://www.iis.net>`_.
 
+Apache configuration
+^^^^^^^^^^^^^^^^^^^^
+
+Here is a virtual host configuration example for ``Apache 2`` web server.
+
+.. warning::
+   Following configuration is only suitable for GLPI version 10.0.7 or later.
```
Comment (cconard96) [inline source/prerequisites.rst:27]:
```suggestion
   The following configuration is only suitable for GLPI version 10.0.7 or later.
```

File: source/prerequisites.rst [hunk 3]
```diff
@@ -16,6 +18,65 @@ GLPI requires a web server that supports PHP, like:
...
+        </Directory>
+    </VirtualHost>
+
+Nginx configuration
+^^^^^^^^^^^^^^^^^^^
+
+Here is a configuration example for ``Nginx`` web server using ``php-fpm``.
+
+.. warning::
+   Following configuration is only suitable for GLPI version 10.0.7 or later.
```
Comment (cconard96) [inline source/prerequisites.rst:53]:
```suggestion
   The following configuration is only suitable for GLPI version 10.0.7 or later.
```

**Review by cconard96 — APPROVED** [review]

**Comment by zoriax** [comment]
Hi,

I'm not sure it's the right place for that but please find here config for IIS redirect 

````xml
<?xml version="1.0" encoding="UTF-8"?>
<configuration>
    <system.webServer>
        <rewrite>
            <rules>
                <rule name="Rewrite to index.php">
					<match url="^(.*)$" />
					<conditions>
						<add input="{REQUEST_FILENAME}" matchType="IsFile" negate="true" />
					</conditions>
					<action type="Rewrite" url="index.php" appendQueryString="true" />
				</rule>
            </rules>
        </rewrite>
    </system.webServer>
</configuration>
``

**Comment by cedric-anne** [comment]
@zoriax 

Could you please open a pull request or an issue, so we could discuss about it?

**Pull request merged**